docs-2.0/3.ngql-guide/1.nGQL-overview/1.overview.md

```diff
@@ -41,101 +44,145 @@ For details of the symbols in nGQL syntax, see the following table:
-
-    Here is an example:
-
-    ```text
-    Feature: Match seek by tag
-
-      Background: Prepare space
-        Given a graph with space named "basketballplayer"
-
-      Scenario: seek by empty tag index
-        When executing query:
-          """
-          MATCH (v:bachelor)
-          RETURN id(v) AS vid
-          """
-        Then the result should be, in any order:
-          | vid          |
-          | 'Tim Duncan' |
-        And no side effects
-        When executing query:
-          """
-          MATCH (v:bachelor)
-          RETURN id(v) AS vid, v.age AS age
-          """
-        Then the result should be, in any order:
-          | vid          | age |
-          | 'Tim Duncan' | 42  |
-        And no side effects
-    ```
-
-    The keywords in the preceding example are described as follows:
-
-    |Keyword|Description|
-    |-|-|
-    |`Feature`|Describes the topic of the current `.feature` file.|
-    |`Background`|Describes the background information of the current `.feature` file.|
-    |`Given`|Describes the prerequisites of running the test statements in the current `.feature` file.|
-    |`Scenario`|Describes the purpose of the scenario. If there is the `@skip` before `Scenario`, this scenario may not work and don't use it as a working example.|
-    |`When`|Describes the nGQL statement to be executed.|
-    |`Then`|Describes the expected result of running the statement in the `When` clause. If the result in your environment does not match the result described in the `.feature` file, submit an [issue](https://github.com/vesoft-inc/nebula-graph/issues) to inform     the Nebula Graph team.|
-    |`And`|Describes the side effects of running the statement in the `When` clause.|
-    | @skip | This test case will be skipped. Commonly, the to-be-tested code is not ready. |
-
-    Welcome to [add more practical scenarios](../15.contribution/how-to-contribute.md) and become a Nebula Graph contributor.
-
-???+ faq "Does it support TinkerPop Gremlin?"
-
-    No. And no plan to support that.
+## About openCypher compatibility
 
-## About Data Model
+### Native nGQL and openCypher
 
-???+ faq "Does Nebula Graph support W3C RDF (SPARQL), or GraphQL?"
+Native nGQL is the part of a graph query language designed and implemented by Nebula Graph.
 
-    No. And no plan to support that.
+OpenCypher is a graph query language maintained by openCypher Implementers Group. 
 
-    Nebula Graph's data model is the property graph, and it is a strong schema system.
+The latest version is openCypher 9. The compatible parts of openCypher in nGQL are called openCypher compatible sentences (short as openCypher).
 
-    It doesn't support rdf.
+!!! Note
 
-    Nebula Graph Query Language does not support `SPARQL` nor `GraphQL`.
+    `nGQL` = `native nGQL` + `openCypher compatible sentences`
+
+### Is nGQL compatible with openCypher 9?
+
+!!! compatibility
+  
+  nGQL is partially compatible with openCypher 9.
```

Comment:
There is a format error. If "nGQL is partially compatible with openCypher 9" is the title, please use：
!!! compatibility "nGQL is partially compatible with openCypher 9."